tests/test_wave_05.py

```diff
@@ -71,29 +71,36 @@ def test_create_goal(client):
         }
     }
 
-@pytest.mark.skip(reason="test to be completed by student")
+# @pytest.mark.skip(reason="test to be completed by student")
 def test_update_goal(client, one_goal):
     pass
     # Act
     # ---- Complete Act Here ----
-
+    response = client.put("/goals/1", json={"title": "MY NEWEST NEWST GOAL"})
+    response_body = response.get_json()
     # Assert
     # ---- Complete Assertions Here ----
-    # assertion 1 goes here
-    # assertion 2 goes here
-    # assertion 3 goes here
+    assert response.status_code == 200
+    assert "goal" in response_body
+    assert response_body == {
+        "goal": {
+            "id": 1,
+            "title": "MY NEWEST NEWST GOAL"
+        }
+    }
```

Comment:
Nice assertions on the response! It's a good idea to also add assertions to make sure that the goal was actually modified in the database, similar to some of the other tests in this file.